doc/installation.md

```diff
@@ -28,7 +28,7 @@ $ chmod a+x kerl
 $ mv kerl /usr/bin
 $ kerl list releases
 $ kerl build 18.2 r18.2 # this takes a while
-$ kerl install r18.2 /usr/lib/erlang
+$ kerl install r18.2 /usr/local/erlang
```

Comment:
I wouldn't recommend to install erlang to /usr using kerl, it will definitely confuse the system admin. It's better to keep it at home.